lib/lighthouse/benefits_documents/service.rb

```diff
@@ -4,6 +4,7 @@
 require 'lighthouse/benefits_documents/configuration'
 require 'lighthouse/service_exception'
 
+# [wipn8923] this is the new document service that delegates to the Lighthouse stuff
```

Comment:
This file is built around the mobile team's use case and didn't fit neatly into ours